setup.py

```diff
@@ -1,4 +1,4 @@
-"""mBuild: A hierarchical, component based molecule builder
+"""mBuild: A hierarchical, component based molecule builder.
```

Comment:
this is an existing issue: Should this be "component-based" instead of "component based "?